WordPress/Classes/ViewRelated/Feature Highlight/TooltipPresenter.swift

```diff
@@ -0,0 +1,81 @@
+import UIKit
+
+final class TooltipPresenter {
+    private enum Constants {
+        static let verticalTooltipDistanceToFocus: CGFloat = 8
+        static let horizontalBufferMargin: CGFloat = 20
+    }
+
+    private let containerView: UIView
+    private let tooltip: Tooltip
+    private let targetView: UIView
+
+    init(containerView: UIView, tooltip: Tooltip, targetView: UIView) {
+        self.containerView = containerView
+        self.tooltip = tooltip
+        self.targetView = targetView
+    }
+
+    func show() {
+        containerView.addSubview(tooltip)
+        tooltip.addArrowHead(toXPosition: arrowOffsetX(), arrowPosition: tooltipOrientation())
+        setUpConstraints()
+    }
+
+    private func setUpConstraints() {
+        tooltip.translatesAutoresizingMaskIntoConstraints = false
+
+        var tooltipConstriants: [NSLayoutConstraint] = [
```

Comment:
Noticed a minor typo here:
```suggestion
        var tooltipConstraints: [NSLayoutConstraint] = [
```
(and the usages below)